pkg/apis/eventing/v1/trigger_lifecycle.go

```diff
@@ -217,3 +220,8 @@ func (ts *TriggerStatus) MarkOIDCIdentityCreatedFailed(reason, messageFormat str
 func (ts *TriggerStatus) MarkOIDCIdentityCreatedUnknown(reason, messageFormat string, messageA ...interface{}) {
 	triggerCondSet.Manage(ts).MarkUnknown(TriggerConditionOIDCIdentityCreated, reason, messageFormat, messageA...)
 }
+
+func (ts *TriggerStatus) MarkOIDCIdentityCreatedNotSupported() {
+	// in case the OIDC feature is not supported, we mark the conditon as true, to not mark the Trigger unready.
+	triggerCondSet.Manage(ts).MarkTrueWithReason(TriggerConditionOIDCIdentityCreated, fmt.Sprintf("%s feature not yet supported for this Broker class", feature.OIDCAuthentication), "")
+}
```

Comment:
We can also think about adding this call in `InitializeConditions` so that we don't need to change every implementation

Reply:
:+1: